packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -25,6 +26,11 @@ const OptionLink = styled(NavLink)`
   ${getOptionStyle}
 `;
 
+const OptionExternalLink = styled.div`
```

Comment:
I think we could use the `BaseLink` as it is done in `v2/SimpleMenu`, wdyt?

```js
const OptionExternalLink = styled(BaseLink)`
  text-decoration: none;
  ${getOptionStyle}
`;
```

Reply:
Ya we can do that. Just a small difference there will be though.
We will not get that style for external link and icon as I have shown in screenshot if we use `BaseLink`.
It will be a shown as just simple link.

I think current implementation with icon and different style is more user friendly though ;)

Reply:
mmh I see - I would say that we should have the same design as the v2 version though, so without external link styling 🤔
if we decide to change this one I guess we would need to update the v2 version?
that would be a design decision for @maevalienard

Reply:
I like it with the icon, I think we understand better what will be the action :) The rest is fine to me. Thanks!

Reply:
thank you @maevalienard 
I guess we should then add the same icon when external links for `v2/SimpleMenu`
wdyt @INNOVATIVEGAMER?

Reply:
Ya, Sure. I will update the PR.